src/javascript/EditPanelContainer/EditPanel/EditPanel.jsx

```diff
@@ -36,10 +40,14 @@ const EditPanel = ({fields, siteInfo, nodeData, formik}) => {
                 />,
                 title: nodeData.displayName,
                 contextModifiers: (
-                    <Badge badgeContent={nodeData.primaryNodeType.displayName}
-                           variant="normal"
-                           color="ghost"
-                    />
+                    <React.Fragment>
```

Comment:
```suggestion
                    <>
```

Reply:
Didn't know this syntax, is that the same thing ?

Reply:
Yes, it's new syntax for fragment declaration, it doesn't support keys or attributes, but I guess we don't need it here.

Link: https://reactjs.org/docs/fragments.html#short-syntax